src/services/tests/user.service.spec.ts

```diff
@@ -270,6 +271,64 @@ describe('UserService', () => {
     });
   });
 
+  describe('getUserSavedItems', () => {
+    it('should return a list with saved post and library ordered by updatedAt', async () => {
+      const mockUserId = mockTestUser.user_id;
+
+      jest.spyOn(prisma.user, 'findUnique').mockResolvedValue({
+        user_savedLibrary: [
+          {
+            library: mockTestLibrarySaved,
+            deletedAt: null,
+          },
+        ],
+        user_savedPost: [
+          {
+            post: mockTestPostSaved,
+            deletedAt: null,
+          },
+        ],
+      } as any);
+
+      jest.spyOn(presignedService, 'getDownloadURL').mockResolvedValue('https://example.com/presigned-url');
+
+      const result = await service.getUserSavedItems(mockUserId);
+
+      expect(Array.isArray(result)).toBe(true);
+      expect(result.length).toBe(2);
```

Comment:
```suggestion
      // eslint-disable-next-line no-magic-numbers
      expect(result.length).toBe(2);
```
Aqui é valido desabilitar o no-magic-numbers do lint